bin/opal-bench-chrome

```diff
@@ -0,0 +1,2 @@
+#!/usr/bin/env bash
+bundle exec ruby exe/opal -Ilib -Rchrome %*
```

Comment:
In bash, `%*` is spelled `"$@"` (with `"`s, very important)

Reply:
done